core/src/main/java/nl/nn/adapterframework/configuration/ConfigurationUtils.java

```diff
@@ -53,7 +53,7 @@
 import nl.nn.adapterframework.core.IbisTransaction;
 import nl.nn.adapterframework.core.SenderException;
 import nl.nn.adapterframework.jdbc.FixedQuerySender;
-import nl.nn.adapterframework.jdbc.JdbcException;
+import nl.nn.adapterframework.dbms.JdbcException;
```

Comment:
Ik zie heel veel bestanden in de diff met alleen maar kleine changes zoals deze package-changes.

Dat maakt het PR misschien groter dan nodig. Zoals ik eerder had proberen uit te leggen, ik had liever gezien dat dit soort veranderingen in package naam nu nog niet in dit PR had gezeten.

Dat kunnen we ook nog een keer recht trekken in een PR dat niets anders bevat dan package naam aanpassingen, want dat gaan we binnenkort toch doen met diverse classes. Dan hebben we 1x een paar PRs waarvan we weten dat er geen enkele andere verandering in zit dan het moven van classes, renamen van packages, etc etc.

Dat maakt het makkelijker om de PRs te reviewen.

@nielsm5 wat denk jij, beter om dit soort kleine changes die heel veel files raken nu even terug te draaien voor dit PR?
Of nu maar laten voor wat het is?